independent-publisher-connectors/ClickUp Team Manager/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1231 @@
+				  "x-ms-summary":"Statuses"
+					  
+                },
+                "lists": {
+                  "type": "array",
+                  "items": {},
+                  "description": "lists",
+				  "x-ms-visibility":"internal"
+					  
+                },
+                "permission_level": {
+                  "type": "string",
+                  "description": "permission_level",
+				  "x-ms-summary":"Permission Level"
+				 
+					  
+                }
+              }
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "id": {
+                  "type": "string",
+                  "description": "id"
+					  
+                },
+                "name": {
+                  "type": "string",
+                  "description": "name"
+					  
+                },
+                "orderindex": {
+                  "type": "integer",
+                  "format": "int32",
+                  "description": "orderindex"
+					  
+                },
+                "override_statuses": {
+                  "type": "boolean",
+                  "description": "override_statuses"
+					  
+                },
+                "hidden": {
+                  "type": "boolean",
+                  "description": "hidden"
+					  
+                },
+                "space": {
+                  "type": "object",
+                  "properties": {
+                    "id": {
+                      "type": "string",
+                      "description": "id"
+					  
+                    },
+                    "name": {
+                      "type": "string",
+                      "description": "name"
+					  
+                    },
+                    "access": {
+                      "type": "boolean",
+                      "description": "access"
+					  
+                    }
+                  },
+                  "description": "space"
+                },
+                "task_count": {
+                  "type": "string",
+                  "description": "task_count"
+					  
+                },
+                "archived": {
+                  "type": "boolean",
+                  "description": "archived"
+					  
+                },
+                "statuses": {
+                  "type": "array",
+                  "items": {},
+                  "description": "statuses"
+					  
+                },
+                "lists": {
+                  "type": "array",
+                  "items": {},
+                  "description": "lists"
+					  
+                },
+                "permission_level": {
+                  "type": "string",
+                  "description": "permission_level"
+                }
+              }
+            }
+          }
+        }
+      }
+    },
+    "/api/v2/folder/folder_id": {},

```

Comment:
@marsurice Empty operation definitions can be removed as they would not add any value to the connector.